File: Manual 01/Question08.cpp
#include <iostream>
using namespace std;

int main()
{
    float a, b, c, avg;

    // Input three numbers
    cout << "Enter the 1st number: ";
    cin >> a;
    cout << "Enter the 2nd number: ";
    cin >> b;
    cout << "Enter the 3rd number: ";
    cin >> c;

    // Check if all numbers are positive
    if (a > 0 && b > 0 && c > 0)
    {
        avg = (a + b + c) / 3;
        cout << "The average of the numbers is: " << avg << endl;
    }
    else
    {
        cout << "Invalid input. All numbers must be positive." << endl;
    }

    return 0;
}

File: Python/Manual01.py/Question08.py
n = int(input("Enter Size of Data: "))

if n <= 0:
    print("Enter a Positive Number")
else:
    newList = list()
    i = 0
    while i < n:
        newList.append(int(input("Enter Value #" + str(i + 1) + ": ")))
        i += 1

    i = 0
    sum = float(0)
    while i < n:
        sum += newList[i]
        i += 1

    print("Sum of Values: " + str(sum) + "\nAverage of Values: " + str(sum / n))
